client/me/concierge/shared/appointment-info.js

```diff
@@ -115,7 +115,7 @@ class AppointmentInfo extends Component {
 							<br />
 							<FormSettingExplanation>
 								{ translate(
-									'Note: You have 30 days from the date of purchase to cancel an unused Quick Start ' +
+									'Note: You have 14 days from the date of purchase to cancel an unused Quick Start ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Note: You have 30 days from the date of purchase to cancel an unused Quick Start session and receive a refund. Please note, if you miss a scheduled session twice, the purchase will be cancelled without a refund.' )` *ES Score: 10*

Reply:
lol